### PR TITLE
Update Repeating Transactions to Scheduled Transactions in mmex.json

### PR DESCRIPTION
**Pull request by ovari**

### CODE REVIEW

**Comment by guanlisheng** [comment]
thanks, @ovari , would we focus on `Scheduled` vs `Repearting` in this PR?

**Comment by ovari** [comment]
Are the others no good?

**Comment by guanlisheng** [comment]
some changes are still good, e.g. Ubuntu removal.   just wanted to make PR topic clear.

**Comment by ovari** [comment]
> some changes are still good, e.g. Ubuntu removal. just wanted to make PR topic clear.

If there are improvements it is easier to just to them as this is all being done in a web-browser.

File: data/features/mmex.json [hunk 1]
```diff
@@ -59,13 +59,13 @@
         },
         {   "name": "crossplatform",
             "icon": "fas fa-desktop",
-            "title": "Cross Platform",
-            "text": "We provide builds for the most common operating systems: Windows, MacOS and Linux Ubuntu. It is possible to use it also on other OS building it directly from source code."
+            "title": "Cross-Platform",
```
Comment (ghost) [inline data/features/mmex.json]: Requires a hyphen.

https://en.wikipedia.org/wiki/Cross-platform_software

File: data/features/mmex.json [hunk 2]
```diff
@@ -59,13 +59,13 @@
...
+            "text": "We provide builds for the most common operating systems: Windows, macOS and Linux. It is possible to use it also on other operating systems building MMEX directly from source code."
         },
         {   "name": "opensource",
             "icon": "fab fa-github",
-            "title": "Free & Open Source",
-            "text": "OpenSource software, free download without ads and charges. It can be freely used, changed, and shared by anyone."
+            "title": "Free and Open-Source",
+            "text": "Open-source software, ad-free and gratis download. It can be freely used, changed, and shared by anyone."
```
Comment (ghost) [inline data/features/mmex.json]: Requires a hyphen.
free as in freedom or free as in money? Both?

https://en.wikipedia.org/wiki/Open-source_software

**Comment by ovari** [comment]
@guanlisheng can you please merge so fork can be deleted and then another merge request sent? Thank you

**Review by guanlisheng — APPROVED** [review]

**Comment by guanlisheng** [comment]
@ovari , can you help to update the URL and screenshot here as well?
https://moneymanagerex.org/docs/features/repeating/

**Comment by ovari** [comment]
> @ovari , can you help to update the URL and screenshot here as well? https://moneymanagerex.org/docs/features/repeating/

@guanlisheng should be ready for you to merge.

**Pull request merged**